bgbb/sql/sql_utils.py

```diff
@@ -144,6 +147,7 @@ def r():
 """
 
 
+# TODO: test both holdout=True and False
```

Comment:
Is this a TODO for this PR or for later?

Reply:
Uhh...later